CHANGELOG.unreleased.md

```diff
@@ -12,6 +12,7 @@
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
 - [Pool] Fix `Headers Timeout Error` when installing patches on XCP-ng
+- [XOSTOR] Remove the limitation of having licensed hosts (PR [#7628](https://github.com/vatesfr/xen-orchestra/pull/7628))
```

Comment:
```suggestion
- [XOSTOR] Don't require host licenses to run XOSTOR (PR [#7628](https://github.com/vatesfr/xen-orchestra/pull/7628))
```

Reply:
Also, is this something we want to communicate about? If the license management is going to change again, maybe we should communicate once it's stable so that it doesn't confuse users?

Reply:
In the short term, no changes are planned.

Licensing will change in the future, if we see that having no restrictions on licensing XCP-ng hosts opens many support tickets